### PR TITLE
17899

### PR DESCRIPTION
**Pull request by anushkamittal2001**
https://nirmata.atlassian.net/browse/NDEV-17899

### CODE REVIEW

**Comment by anushkamittal2001** [comment]
@pns-nirmata this PR is ready!

File: charts/nirmata/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 type: application
 name: kyverno
 version: 3.0.5-rc6
```
Comment (pns-nirmata) [inline charts/nirmata/Chart.yaml:4]: This needs to be increased

File: charts/nirmata/templates/admission-controller/role.yaml [hunk 2]
```diff
@@ -26,9 +26,18 @@ rules:
       - get
       - list
       - watch
+      - create
+      - delete
+      - update
     resourceNames:
       - {{ include "kyverno.config.configMapName" . }}
       - {{ include "kyverno.config.metricsConfigMapName" . }}
```
Comment (pns-nirmata) [inline charts/nirmata/templates/admission-controller/role.yaml:34]: Need one more resource name, that of the success configmap right?

File: charts/nirmata/values.yaml [hunk 3]
```diff
@@ -1619,6 +1619,8 @@ licenseManager:
   callHomeServer: nirmata.io
   # -- License key
   licenseKey: free-tier-license
+    # -- License secret containing license string under key 'license'
+  licenseSecret:
```
Comment (pns-nirmata) [inline charts/nirmata/values.yaml:1623]: Linter error, remove trainlng spaces in this key.

**Review by pns-nirmata — APPROVED** [review]

**Pull request merged**